.github/workflows/rails-new-docker.yml

```diff
@@ -15,9 +15,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
-    - uses: ruby/setup-ruby@v1
+    - name: Set up Ruby 3.3
```

Comment:
Why add this new named step?

Reply:
Its for making consistent across all the workflows.